src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -394,8 +394,8 @@ public void update(double time, double dt) {
                 if (getAntiTip() && driveSignal != null) {
                     signal = new HolonomicDriveSignal( // create updated drive signal
                             accelLimiter.calculate( // vector accel limiter
-                                    driveSignal.getTranslation().rotateBy(driveSignal.isFieldOriented() ? // flatten
-                                            getAngle() : Rotation2.ZERO)) // same code as other block
+                                    driveSignal.getTranslation()).rotateBy(driveSignal.isFieldOriented() ? // flatten
+                                            getAngle() : Rotation2.ZERO) // same code as other block
```

Comment:
Nit: Please move `driveSignal.getTranslation()` to the vector accel limiter line for clarity.